### ISSUE TITLE
ImportError in Python 3.9 environments

### ISSUE THREAD

**Issue by tomkralidis** [elastic/elasticsearch-py#1857]
**Elasticsearch version** (`bin/elasticsearch --version`): 7.5.2

**`elasticsearch-py` version (`elasticsearch.__versionstr__`)**: 7.0.2

Please make sure the major version matches the Elasticsearch server you are running.

**Description of the problem including expected versus actual behavior**:
Import error on Python 3.9 environments.

**Steps to reproduce**:

```python
    from elasticsearch import Elasticsearch, logger as elastic_logger
/opt/hostedtoolcache/Python/3.9.9/x64/lib/python3.9/site-packages/elasticsearch-7.0.2-py3.9.egg/elasticsearch/__init__.py:24: in <module>
    from .client import Elasticsearch
/opt/hostedtoolcache/Python/3.9.9/x64/lib/python3.9/site-packages/elasticsearch-7.0.2-py3.9.egg/elasticsearch/client/__init__.py:4: in <module>
    from ..transport import Transport
/opt/hostedtoolcache/Python/3.9.9/x64/lib/python3.9/site-packages/elasticsearch-7.0.2-py3.9.egg/elasticsearch/transport.py:4: in <module>
    from .connection import Urllib3HttpConnection
/opt/hostedtoolcache/Python/3.9.9/x64/lib/python3.9/site-packages/elasticsearch-7.0.2-py3.9.egg/elasticsearch/connection/__init__.py:2: in <module>
    from .http_requests import RequestsHttpConnection
/opt/hostedtoolcache/Python/3.9.9/x64/lib/python3.9/site-packages/elasticsearch-7.0.2-py3.9.egg/elasticsearch/connection/http_requests.py:3: in <module>
    from base64 import decodestring
E   ImportError: cannot import name 'decodestring' from 'base64' (/opt/hostedtoolcache/Python/3.9.9/x64/lib/python3.9/base64.py)
```

**Comment by sethmlarson** [comment]
[v7.0.2 of elasticsearch-py](https://pypi.org/project/elasticsearch/7.0.2/) was released in May of 2019 and Python 3.9 was released in October of 2020 meaning that version of the package couldn't have had explicit support for Python 3.9 yet as it was not yet released (Python 3.8 hadn't been released yet either!) You'll have to upgrade to a more recent version of the client for Python 3.9 support, try upgrading to the latest version of 7.x.

Closing this for now.

**Comment by tomkralidis** [comment]
Works, thanks much for the recommendation.
